src/UI/Components/Diagram/components/FormModal.tsx

```diff
@@ -80,7 +76,7 @@ const FormModal = ({
       onClick={(val) => setIsSelectOpen(val)}
       isExpanded={isSelectOpen}
       aria-label="service-picker"
```

Comment:
service-picker?

Reply:
like above, form modal is in the process, more like it will be gone, but I didn't want to touch it here as there is separate ticket